parse/src/test/java/com/parse/AlgorithmParameterSpecExt.kt

```diff
@@ -0,0 +1,22 @@
+package com.parse
+
+/*
+ * Copyright 2020 Appmattus Limited
```

Comment:
We need to make sure to consider the license implications. This repository is planned to be migrated to Apache 2, we should do that before merged this, otherwise we need to add this as a second license.